civis/ml/tests/test_model.py

```diff
@@ -67,53 +68,6 @@
 }
 
 
-def setup_client_mock(script_id=-10, run_id=100, state='succeeded',
-                      run_outputs=None):
```

Comment:
Removing this mock client creator in favor of the newer one called `create_client_mock_for_container_tests` from #416